components/Common/ProgressionSidebar/ProgressionSidebarGroup/index.module.css

```diff
@@ -0,0 +1,49 @@
+.group,
+ul {
+  @apply flex
+    list-none
+    flex-col
+    p-0;
+}
+
+.group {
+  & > li {
+    @apply flex
+      flex-col
+      gap-4
+      text-sm
+      font-medium
+      text-neutral-800
+      dark:text-neutral-200;
+  }
+
+  .items {
+    @apply relative
+      -left-1
+      gap-2
+      shadow-[inset_7px_0_0_theme(colors.white),_inset_8px_0_0_0_theme(colors.neutral[200])]
+      dark:shadow-[inset_7px_0_0_theme(colors.neutral[950]),_inset_8px_0_0_0_theme(colors.neutral[800])];
+  }
+
+  .items li {
+    &:first-child {
+      @apply before:absolute
```

Comment:
Isn't better just &:before {} and apply the styles inside? Just for separation? idk